### PR TITLE
Resolve #1689 DAI incorrect network fee

### LINKED ISSUE #1689: DAI incorrect network fee

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/actions/transferActions.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ function submitDirectTransferTx (txRequest: {
   transferAmount: StandardTokenUnit,
   transferFiatAmountSpot: string,
   fiatType: string,
+  exchangeRate: { cryptoExchangeRate: string, txFeeCryptoExchangeRate: string },
```
Comment (InVincible2016) [inline src/actions/transferActions.js:53]: place exchangeRate definition in .flow file

File: src/actions/transferActions.js [hunk 2]
```diff
@@ -597,9 +605,10 @@ async function _getTransfer (transferId: ?string, receivingId: ?string) {
   transferData.txFee = await _getTxFeeForTransfer(transferData)
 
   if (transferData.txFee) {
+    transferData.txFee.costInStandardUnit = '0.003861872'
```
Comment (InVincible2016) [inline src/actions/transferActions.js:608]: Why is this a fixed constant?

File: src/actions/transferActions.js [hunk 3]
```diff
@@ -259,6 +266,7 @@ async function _transactionHashRetrieved (txRequest: {|
   transferAmount: StandardTokenUnit,
   transferFiatAmountSpot: string,
   fiatType: string,
+  exchangeRate: { cryptoExchangeRate: string, txFeeCryptoExchangeRate: string },
```
Comment (InVincible2016) [inline src/actions/transferActions.js:269]: Will need to make changes to the backend as well?

File: src/actions/transferActions.js (hunk 2 above)
Comment (timothywangdev) [reply src/actions/transferActions.js:608]: should be  removed

File: src/actions/transferActions.js (hunk 3 above)
Comment (timothywangdev) [reply src/actions/transferActions.js:269]: Will need to make changes to the backend as well?

Comment (timothywangdev) [reply src/actions/transferActions.js:269]: already pushed in previous backend pr

File: src/actions/transferActions.js (hunk 1 above)
Comment (timothywangdev) [reply src/actions/transferActions.js]: fixed

**Pull request merged**